src/handlers/baseFieldsHandlers.ts

```diff
@@ -83,8 +110,123 @@ const putBaseField = (
 		});
 };
 
+const getBaseFieldLocalizationsByBaseFieldId = (
+	req: Request<{ baseFieldId: string }>,
+	res: Response,
+	next: NextFunction,
+): void => {
+	const baseFieldId = Number.parseInt(req.params.baseFieldId, 10);
+	if (Number.isNaN(baseFieldId)) {
+		next(new InputValidationError('The entity id must be a number.', []));
+		return;
+	}
+	Promise.all([assertBaseFieldExistsForLocalization(baseFieldId)])
```

Comment:
Since you're just working with a single promise you don't need to use `Promise.all` but rather can just call the assertion directly.